agent/mibgroup/hardware/fsys/fsys_mntent.c

```diff
@@ -176,7 +179,240 @@ _fsys_type( char *typename )
 void
 netsnmp_fsys_arch_init( void )
 {
-    return;
+    snmpd_register_config_handler("ignoremount", parse_mount_config,
+                                  free_mount_config, "name");
```

Comment:
This is the wrong location to add an snmpd_register_config_handler() call. The file in which this call has been added, fsys_mntent.c, implements architecture support for Linux but it not used on e.g. FreeBSD systems. Please move this call into init_hw_fsys().